app/commands/track/retrieve_maintainers_status.rb

```diff
@@ -0,0 +1,90 @@
+class Track::RetrieveMaintainersStatus
```

Comment:
This is a relatively expensive command. What's the best way to cache this data? Redis?

Reply:
See below